app/src/AddJoke.tsx

```diff
@@ -1,27 +1,78 @@
-import React, { useState } from "react";
+import { useState, ChangeEvent } from "react";
+import { AppliedTheFavouriteJoke } from "./AppliedTheFavouriteJoke";
+import { IJokes } from "./interfaces";
 
-export default function AddJoke() {
-  const [joke, setAddJoke] = useState("");
-  const submitForm = (event: React.FormEvent<HTMLFormElement>) => {
+interface AddJokeProps {
+  status: string;
+}
+
+export default function AddJoke({ status = "empty" }: AddJokeProps) {
+  const [joke, setJoke] = useState<string>("");
+  const [pun, setPun] = useState<string>("");
+  const [jokeList, setJokeList] = useState<IJokes[]>([]);
+
+  // conditional statement to set status to 'success' if user submits correct answer
+  if (status === "success") {
+    return <h1>That's right!</h1>;
+  }
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>): void => {
+    event.preventDefault();
+    if (event.target.name === "joke") {
```

Comment:
Very clear conditional stating that its the event targets name is equal to joke then set that current value of the submitted joke, else set the pun values, a very effective way 🌈 🎲